code/index.html

```diff
@@ -4,19 +4,90 @@
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin> 
+<link href="https://fonts.googleapis.com/css2?family=IBM+Plex+Sans+KR:wght@100&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+
+    <header class="header">
+       <img src="https://i.ibb.co/k3ykDzt/LOGO.jpg" alt="Logo">
+       <div class="logo-name">News site project</div>
+        <nav class="navigation">
+             <a href="https://www.google.com/?client=safari">World News</a>
+             <a href="https://www.google.com/?client=safari">Weather</a>
+             <a href="https://www.google.com/?client=safari">Entertainment</a>
+             <a href="https://www.google.com/?client=safari">Contact</a>
+        </nav>
     </header>
 
     <section class="big-news">
-      <!-- The big news content -->
+       <img class="big-news-image" src="https://i.picsum.photos/id/1036/4608/3072.jpg?hmac=Tn9CS_V7lFSMMgAI5k1M38Mdj-YEJR9dPJCyeXNpnZc" alt="image of plants"/>
+        <p class="big-news-text">Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source. Lorem Ipsum comes from sections 1.10.32 and 1.10.33 of "de Finibus Bonorum et Malorum" (The Extremes of Good and Evil) by Cicero, written in 45 BC. This book is a treatise on the theory of ethics, very popular during the Renaissance.The first line of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes from a line in section 1.10.32. The standard chunk of Lorem Ipsum used since the 1500s is reproduced below for those interested. Sections 1.10.32 and 1.10.33 from "de Finibus Bonorum et Malorum" by Cicero are also reproduced in their exact original form, accompanied by English versions from the 1914 translation by H. Rackham. It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).
+        </p>
+       <button>Read more</button>
     </section>
 
-    <section class="">
-      <!-- The other news  -->
+    <div class="sb sb-3">
+      <hr class="section-break-3"/>
+    </div>
```

Comment:
Instead us using a section break, you could also add a border to the bottom of your header.